Formula/gnupg.rb

```diff
@@ -49,11 +49,19 @@ def install
     system "make"
     system "make", "check"
     system "make", "install"
+
+    on_macos do
+      (buildpath/"scdaemon.conf").write <<~EOS
+        disable-ccid
+      EOS
+      (etc/"gnupg").install "scdaemon.conf"
```

Comment:
```suggestion
      (pkgetc/"scdaemon.conf").write <<~EOS
        disable-ccid
      EOS
```
I think this gets things done quicker by writing to `etc/gnupg/scdaemon.conf` directly. We could also specify `etc/"gnupg/scdaemon.conf"` in case the formula name changes. (`pkgetc == etc/name` where `name` is the name of the formula.)

Reply:
Cool! I just copied this from git's formula. 😄 

Question though, will writing to pkgetc directly resort to {file}.default if the target already exists? (ie, not truncate or append?)

My assumption is that the *.default behavior was handled by the `.install` approach.

Reply:
Ah, yes, you're right. `write` invokes `Pathname#write`, which clobbers existing files. The `brew` extension `Pathname#install` won't clobber existing files when installing into `etc`. Let's stick with what you're doing here.

...maybe add a comment for those of us not paying attention and wondering about indirection here.

Reply:
👍 anything else worth commenting? reference back to the gnupg ticket about why this is necessary or anything?

Reply:
Yep, a link to the gnupg ticket would be good.